test/powershell/Modules/Microsoft.PowerShell.Utility/Clear-Variable.Tests.ps1

```diff
@@ -114,12 +114,6 @@ Describe "Clear-Variable" -Tags "CI" {
 	{ Get-Variable var1 } | Should Not Throw
 		}
 
-		It "Should work using the clv alias" {
-	clv -Name var1
-	$var1 | Should BeNullOrEmpty
-	{ Get-Variable var1 } | Should Not Throw
-		}

```

Comment:
This test is not checking if the alias exists, but actually testing the functionality of `clear-variable`, so it shouldn't be removed.